### DISCUSSION TITLE
Overlay Date and Time stamp issue- wrong date

### DISCUSSION THREAD

**Discussion by Knicks325** [gopro/labs#982] [Q&A]
I recoded some videos today that contain the date and timestamp overlay- I usually never have a problem and it worked correctly yesterday. However, after I opened the videos I see instead of having today's Date overlay of 7/9/24 it has the date 4/19/24 -- is it possible to fix this issue for me to somehow change the date stamp on the video

**Comment by dnewman-gpsw** [comment]
It is part of the video, not fixable.  When using overlay you should regularly set the date and time, or confirm that they are correct, particularly if the battery is flat, or removed for a long period.
